hazelcast/src/test/java/com/hazelcast/spi/impl/operationexecutor/slowoperationdetector/SlowOperationDetectorAbstractTest.java

```diff
@@ -113,10 +101,6 @@ static void shutdownOperationService(HazelcastInstance instance) {
         return invocationMap.values();
     }
 
-    static InternalOperationService getInternalOperationService(HazelcastInstance instance) {
-        return TestUtil.getNode(instance).nodeEngine.getOperationService();
-    }
-
     static int getDefaultPartitionId(HazelcastInstance instance) {
```

Comment:
I find default partition id extremely confusing. Also 'default key' is very confusing.